niso-jats.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "lutaml-model"
   spec.add_runtime_dependency "nokogiri"
-  spec.add_dependency "zeitwerk", "~> 2.7"
+  spec.add_runtime_dependency "zeitwerk", "~> 2.6"
```

Comment:
Should be something like `~> 2.6.18`

Reply:
Agree with @Novtopro here, as this lines means zeitwerk "2.7" which is Ruby 3.2+ is also applicable.